app/services/document_publishing_service.rb

```diff
@@ -40,6 +49,14 @@ def payload(document)
   end
 
   def publishing_api
+    GdsApi::PublishingApi.new(
```

Comment:
This should be nicer after https://github.com/alphagov/gds-api-adapters/pull/841 is released.